Serilog.Sinks.Network.Test/UDPTests.cs

```diff
@@ -83,7 +83,7 @@ public void Dispose()
             _listener.Stop();
         }
     }
-    
+
     public class UDPListener

```

Comment:
Only a little thing but let's either move this into its own file or move the TCPServer class back into the TCP tests